### PR TITLE
Cylance playbook update

### PR DESCRIPTION
**Pull request by adi88d**
## Status
Ready

## Related Issues
fixes: Update playbook to support Cylance Protect v2 and Cylance Protect v1 integrations

## Description
'Get File Sample By Hash - Cylance Protect' playbook was able to works with only with the old Cylance Protect integration, now it works with the old Cylance Protect and the new Cylance Protect(v2)

### CODE REVIEW

**Comment by Itay4** [comment]
@adi88d Does this fix some issue?

**Comment by yardensade** [comment]
> @adi88d Does this fix some issue?

The playbook had to support cylance v2, was not an option until I implemented the download-threat for v2

**Comment by adi88d** [comment]
> > @adi88d Does this fix some issue?
> 
> The playbook had to support cylance v2, was not an option until I implemented the download-threat for v2
> 
> @adi88d what are we waiting for?

Review from @anara123

**Comment by yardensade** [comment]
Seems like it does not pass on circle and fails on file structure validation.
@anara123 @ronykoz @adi88d

File: Tests/id_set.json [hunk 1]
```diff
@@ -15264,6 +15259,17 @@
...
+        {
+            "Get File Sample By Hash - Generic - Test": {
+                "name": "Get File Sample By Hash - Generic - Test", 
+                "implementing_scripts": [
+                    "DeleteContext", 
+                    "VerifyContext", 
+                    "Set"
+                ], 
+                "implementing_playbooks": [
+                    "Get File Sample By Hash - Generic"
```
Comment (ronykoz) [inline Tests/id_set.json]: You probably touched this file and removed the `]`

File: Tests/id_set.json [hunk 2]
```diff
@@ -15276,6 +15282,17 @@
...
+        {
+            "Get File Sample From Hash - Generic - Test": {
+                "name": "Get File Sample From Hash - Generic - Test", 
+                "implementing_scripts": [
+                    "DeleteContext", 
+                    "VerifyContext", 
+                    "Set"
+                ], 
+                "implementing_playbooks": [
+                    "Get File Sample From Hash - Generic"
```
Comment (ronykoz) [inline Tests/id_set.json]: Same here

File: Playbooks/playbook-Get_File_Sample_From_Hash_-_Generic.yml [hunk 3]
```diff
@@ -2,83 +2,114 @@ id: Get File Sample From Hash - Generic
 version: -1
 name: Get File Sample From Hash - Generic
 toversion: 3.1.0
-fromversion: ""
+fromversion: 0.0.0
```
Comment (ronykoz) [inline Playbooks/playbook-Get_File_Sample_From_Hash_-_Generic.yml]: not needed

**Comment by ronykoz** [comment]
@yardensade take a look at the id_set file here, it basically showed where the file was corrupted(Painted the brackets in red)

File: Tests/conf.json [hunk 4]
```diff
@@ -992,7 +992,7 @@
         "Lastline": "Out of quota",
         "Netskope": "instance is down",
         "Joe Security": "Reached number of allowed submissions per month, untill 1/2/19",
-        "VirusTotal": "Reached number of allowed submissions"
+        "VirusTotal - Private API": "Reached number of allowed submissions"
```
Comment (yardensade) [inline Tests/conf.json]: Please try to stick to the branch topic, skipping failing tests should be done from a different branch and then pull from master in this branch.

Comment (adi88d) [reply Tests/conf.json]: So if I want to skip some tests, I need to change it only in master branch?

Comment (yardensade) [reply Tests/conf.json]: If you want to skip a test, open a branch for it, skip it, merge to master, then pull from master in this branch.

Comment (anara123) [reply Tests/conf.json]: @adi88d please unskip the tests, and lets merge

**Pull request merged**